src/rust/ide/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -39,12 +39,13 @@ use ensogl_theme as theme;
 // === Constants ===
 // =================
 
-const DEFAULT_SIZE      : (f32,f32) = (200.0,200.0);
-const CORNER_RADIUS     : f32       = super::super::node::CORNER_RADIUS;
-// Note[mm]: at the moment we use a CSS replacement shadow defined in the .visualization class of
-// `src/js/lib/content/src/index.html`. While that is in use this shadow is deactivated.
-const SHADOW_SIZE       : f32       = 0.0 * super::super::node::SHADOW_SIZE;
-const ACTION_BAR_HEIGHT : f32       = 2.0 * CORNER_RADIUS;
+/// Default width and height of the visualisation container.
+pub const DEFAULT_SIZE      : (f32,f32) = (200.0,200.0);
+    const CORNER_RADIUS     : f32       = super::super::node::CORNER_RADIUS;
+    // Note[mm]: at the moment we use a CSS replacement shadow defined in the .visualization class of
+    // `src/js/lib/content/src/index.html`. While that is in use this shadow is deactivated.
+    const SHADOW_SIZE       : f32       = 0.0 * super::super::node::SHADOW_SIZE;
+    const ACTION_BAR_HEIGHT : f32       = 2.0 * CORNER_RADIUS;
```

Comment:
according to style docs we are not alignment on `pub` this way. I dont know TBH what to think about it - from one hand it looks cool. On the other hand, it is confusing when tyour screen is scrolled this way, that you dont see the first line and you only see indented lines. Until we talk about it, I'd propose not to do this indentation.